invenio.cfg

```diff
@@ -185,12 +185,13 @@ from invenio_records_resources.services.custom_fields import (
     DoubleCF,
     ISODateStringCF,
     KeywordCF,
+    TextCF,
 )
 from marshmallow import fields
 from marshmallow_utils.fields import SanitizedUnicode
 
 
-class RelationshipListCF(BaseListCF):
+class RelationshipListCF(BaseCF):
```

Comment:
neat: maybe you can start using the `site` folder containing the instance's custom code for better readability... I had a [wip PR](https://github.com/zenodo/zenodo-rdm/pull/61) regarding adding the support view.